xds/src/main/java/io/grpc/xds/WeightedTargetLoadBalancer.java

```diff
@@ -17,18 +17,22 @@
 package io.grpc.xds;
 
 import static com.google.common.base.Preconditions.checkNotNull;
+import static io.envoyproxy.envoy.service.discovery.v2.AggregatedDiscoveryServiceGrpc.getStreamAggregatedResourcesMethod;
```

Comment:
Why this is introduced? This seems to be completely wrong...

Reply:
oh i was leveraging any `methodDescriptor` to do a dummy pick, not supposed to be useful, i know it was probably too error prone. ejona@ seems to have a better idea.